_data/chains/eip155-67390.json

```diff
@@ -0,0 +1,22 @@
+
+{
+  "name": "SiriusNet",
+  "chain": "SIRIUS",
+  "rpc": ["https://u0tnafcv6j:o2T045sxuCNXL878RDQLp5__Zj-es2cvdjtgkl4etn0@u0v7kwtvtg-u0wj114sve-rpc.us0-aws.kaleido.io/"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "MCD",
+    "symbol": "MCD",
+    "decimals": 18
+  },
+  "infoURL": "https://macaucasinolisboa.xyz",
+  "shortName": "mcl",
+  "chainId": 67390,
+  "networkId": 67390,
+
+  "explorers": [{
+    "name": "siriusnetscan",
+    "url": "https://siriusnetscan.io",
```

Comment:
seems to be down
![image](https://user-images.githubusercontent.com/111600/196562818-09ed52e9-c3a4-486d-bc89-22490155588e.png)

Reply:
https://u0tnafcv6j:o2T045sxuCNXL878RDQLp5__Zj-es2cvdjtgkl4etn0@u0v7kwtvtg-u0ju2g502d-firefly-os.us0-aws-ws.kaleido.io/ui